problem1.cpp

```diff
@@ -1,3 +1,20 @@
+// Time Complexity  : O(n) --> We iterate through the entire linked list
+// Space Complexity : O(n) --> We create a new linked list (current) that reverses all the values from the previous linked list(head)
```

Comment:
Think about this again

Reply:
Thank you for pointing this out Ashmeet. I had misunderstood it to be O(N) instead of O(1). Got it clarified after yesterday's lecture. I will update my pull request

Reply:
Good Gaurav